### ISSUE TITLE
TF Plan security: Terragrunt flow - full JSON in output

### ISSUE THREAD

**Issue by chvima** [diggerhq/digger#1248]
Using v 0.4.6.

For project that use Terragrunt, during digger plan, Digger CLI is outputting full plan JSON (as if I would use terraform show -json plan.tf - https://developer.hashicorp.com/terraform/internals/json-format ). This is a security issue for us as it also includes sensitive values (we are generally not trying to avoid having sensitive values in plan as the plan is stored in secured s3 backend).

Using Terraform flow, the full plan is not outputted.

Snippet from the log of job:
```
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create
  - destroy
...
Terraform will perform the following actions:
Plan: 4 to add, 0 to change, 4 to destroy.

{"format_version":"1.2","terraform_version":"1.5.7","variables"
...
"timestamp":"2024-03-04T15:58:12Z"}
no $DIGGER_OUT file at: /__w/_temp/digger-out.log
No plan policies found, succeeding
Digger finished successfully
```

Can this output be removed from the digger plan flow when using terragrunt?

**Comment by motatoes** [comment]
addressed in #1475

**Comment by chvima** [comment]
thanks!
